pxtlib/docsrender.ts

```diff
@@ -275,10 +278,17 @@ namespace pxt.docs {
             params["github"] = "";
         }
 
+        // Add accessiblity menu 
+        const accMenuHtml = `
+            <a href="#maincontent" class="ui item link" tabindex="0" role="menuitem">${lf("Skip to main content")}</a>
```

Comment:
It's for the documentation. There is a hidden menu that appear when you start to tab into the documentation that let you skip the side menu and tab directly into the content.